src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumeSQLRepository.cs

```diff
@@ -16,33 +17,60 @@ public class ResumeSQLRepository : IResumeSQLRepository
 
         public ResumeSQLRepository()
         {
+            IConfigurationRoot config = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("local.settings.json", optional: false, reloadOnChange: true)
+            .Build();
             this.connectionString = Environment.GetEnvironmentVariable("SQLConnetionString", EnvironmentVariableTarget.Process);
         }
 
         public Resume Add(Resume newObject)
         {
-            throw new NotImplementedException();
+            var sql = "INSERT INTO Resume ( Id, Title, Sumary, CreationDate, LastUpdate ) VALUES (@id, @title, @sumary, @creationdate, @lastupdate)";
+            using (IDbConnection db = new SqlConnection(this.connectionString))
+            {
+                db.Open();
+                DynamicParameters parameter = new DynamicParameters();
+                parameter.Add("@id", newObject.Id);
+                parameter.Add("@title", newObject.Title);
+                parameter.Add("@sumary", newObject.Summary);
+                parameter.Add("@creationdate", DateTime.Now, DbType.DateTime, ParameterDirection.Input);
+                parameter.Add("@lastupdate", DateTime.Now, DbType.DateTime, ParameterDirection.Input);
+                db.Execute(sql, parameter);
+            }
+
+            return newObject;
         }
 
         public void Delete(Guid id)
         {
             throw new NotImplementedException();
         }
 
-        public List<Resume> GetAll()
+        public IEnumerable<Resume> GetAll()
         {
-            List<Resume> resumes = new List<Resume>();
+            IEnumerable<Resume> resumes = new List<Resume>();
             using (IDbConnection db = new SqlConnection(this.connectionString))
             {
-                resumes = db.Query<Resume>("select * from resume").ToList();
+                resumes = db.Query<Resume>("SELECT Id, Title, Sumary, CreationDate, LastUpdate FROM Resume").ToList();
```

Comment:
since it's IEnumerable no conversion to list needed

Reply:
@paulo-teran-jalafund do you agree?

Reply:
Done